src/AsyncEventSource.cpp

```diff
@@ -25,114 +25,97 @@
 
 using namespace asyncsrv;
 
-static String generateEventMessage(const char* message, const char* event, uint32_t id, uint32_t reconnect) {
-  String ev;
+static std::string generateEventMessage(const char* message, const char* event, uint32_t id, uint32_t reconnect) {
+  //String ev;
+  std::string str;
```

Comment:
why ?

Reply:
you mean commented ev? just a leftover... will remove it

Reply:
I meant why changing from String to sd:string: the project is using String everywhere. Let's keep it consistent  ;-)